docs/motivating_use_case.md

```diff
@@ -161,9 +161,9 @@ appropriate party; we only show the claim here.
 
 ```json
 {
-  "attestation_type": "https://example.com/CodeReview/v1",
-  "subject": { "git_commit": "5215a97a7978d8ee0de859ccac1bbfd2475bfe92" },
-  "details": {
+  "predicateType": "https://example.com/CodeReview/v1",
+  "subject": [{ "git_commit": "5215a97a7978d8ee0de859ccac1bbfd2475bfe92" }],
```

Comment:
nit: I know this is a `predicateType` that doesn't exist but since we do have a `gitCommit` type, can we use that here?

```suggestion
  "subject": [{ "digest": {"gitCommit": "5215a97a7978d8ee0de859ccac1bbfd2475bfe92" } }],
```

Reply:
Yeah, I should have done that.  Thanks!